frontend/src/app/modules/posts/components/edit-comment/edit-comment.component.css

```diff
@@ -0,0 +1,21 @@
+.grow-wrap {
```

Comment:
jak to są te same klasy co w `edit-post-component` to należałoby to jakoś wyodrębnić żeby nie mieć 2 razy tego samego kodu.

Reply:
faktycznie